docker/update-permissions.sh

```diff
@@ -2,10 +2,9 @@
 
 set -eo pipefail
 
-zfs_mod="${ZFS_MOD:-g+w}"
-chmod_bin=${ZFS_CHOWN_BIN:-sudo -H chmod}
+chmod_bin=${ZFS_CHMOD_BIN:-sudo -H chmod}
```

Comment:
I know I named it incorrectly, but renaming the Env var would be a breaking change for those people that already override it.
Maybe we can keep the old one as an alias instead, so both work?